tests/Teams/LinkUnfurling/TeamsAppManifest/manifest.json

```diff
@@ -54,7 +54,7 @@
           "type": "link",
           "value": {
             "domains": [
-              "tzadik.com"
+              "*.com"
```

Comment:
can we have a site that's not everything? for example, "microsoft.com" and something else...